.github/workflows/macos.yml

```
@@ -449,7 +449,7 @@ jobs:
       - name: 'install autotools'
         if: ${{ matrix.build == 'autotools' }}
         run: |
-          echo autoconf automake libtool openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+          echo autoconf automake libtool openssl libpsl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
```

```suggestion
          echo autoconf automake libtool openssl pkg-config libpsl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
```
Perhaps this, based on how it's done in the autotools and cmake jobs.

---

Nope, works without `pkg-config`:
https://github.com/curl/curl/actions/runs/10237324011/job/28320483945